### ISSUE TITLE
Windows Server 2016 - remote access does not work on logon screen

### ISSUE THREAD

**Issue by mythral0** [jmdaweb/NVDARemoteServer#32]
Hi, really strange, on windows Server 2016, I cannot remotely control the server on the logon screen, NVDA isn't connecting to the NVDA remote Server for some strange reason.
Once I have logged-in, NVDA then connects to the NVDA Remote Server without any issue.
I have already set:-

Play sounds when starting or exiting NVDA  check box  checked  Alt+p
Logging level:  combo box  info  collapsed  Alt+o
Start NVDA after I sign in  check box  checked  Alt+a
Use NVDA during sign-in (requires administrator privileges)  check box  checked
[Use currently saved settings during sign-in and on secure screens (requires administrator privileges)  button] I have done this from the server itself

still no joy. I've tried it several times.
this works perfectly on Windows 10.

Thus, is there some specific setting on windows Server 2016 which I need to enable for this to work?

Many thanks.

Mo.

**Comment by jmdaweb** [comment]
Hi,
Do you see a dialog with an error message when trying to connect, or you simply hear the connection tones and you can't control anything after that? In the second case, the problem is in the remote machine you are controlling, this isn't a server problem. If the remote NVDA starts automatically on the logon screen, you could try sending a control+alt+del key press from the NVDA Remote menu. This is one difference between Windows 10 and Windows Server. Also, if you setup your server using Windows Remote Desktop, choose logout instead of disconnect when you finish the remote session. Is the NVDA Remote Server installed as a Windows service?
Regards.

**Comment by mythral0** [comment]
Hi José,

Many thanks for your response.

I’ve actually got it working. What I had to do on the server was to disable the ctrl+alt+delete so now it prompts me for the password.

 

However, when the windows server joins the NVDA remote Server when it is on the logon screen, I don’t get the audible tones on the client machine which is already connected to the NVDA remote Server using the same pin, hence the reason I thought that the windows server wasn’t actually connecting to the NVDA Remote Server at the logon screen.

 

The strange thing is once I’ve logged on and the server reconnects to the NVDA Remote Server, I get the audible tones on the client machine in this scenario.

I also have a windows 10 machine that I access remotely and on that I get the audible tones in both situations, on the logon screen and also after logging in.

 

Regarding the ctrl+alt+delete function in the remote menu on the remote server,, for me it is always greyed out. The only time I can press this is on the local machine and when I do this, it does nothing. So I’m not sure how this is supposed to work.

 

Many thanks for all your help.

 

KND RGS

 

Mo.

**Comment by jmdaweb** [comment]
Hi,
The control+alt+del function located inside the NVDA Remote menu is intended to be used from the local machine. Control+alt+del is a key combination reserved for the operating system. You can't change its behaviour with any program. So if you press control+alt+del while you are in a remote session, your machine intercepts it. Thush, if you want to send control+alt+del to the remote machine, you must use this option.
I have no idea about why the tones are played twice on Windows 10 and once on Windows Server, but take into account that there are two copies of NVDA running, two clients: the logon screen client and the session client.
I will mark this issue as closed.
Regards.

**Comment by mythral0** [comment]
José,

 

Unfortunately, the control+alt+del function located inside the NVDA Remote menu just doesn’t work for me.

This is what I did:-

I switch from controlling the remote machine to controlling the local machine.

I activate the control+alt+del function from the remote menu on the local machine.

I see nothing changing on the remote machine.

 

What am I doing wrong?

 

Many thanks.

 

KND RGS

 

Mo.

**Comment by jmdaweb** [comment]
Hi,
This is a problem related to NVDA Remote itself, so you should ask the add-on developers directly.
Visit https://github.com/nvdaremote/nvdaremote/issues and open a new issue.
Regards.

**Comment by mythral0** [comment]
José,

 

Yes, your absolutely correct, I shall do that.

Many thanks for all your help, much appreciated.

 

Many thanks.

 

KND RGS

 

Mo.
